### DISCUSSION TITLE
ReplayGain in m4a/mp4

### DISCUSSION THREAD

**Discussion by Lascaille** [audacious-media-player/audacious#84] [Q&A]
Okay, how exactly do I format replaygain information in mp4/m4a files to make Audacious pick it up?
Because ffmpeg tells me that it's there, mp3tag tells me that it's there, winamp reads it, audacious manages to read the relevant metadata but doesn't get the replaygain info.
I'm using Picard, mostly, to tag and organise these files.

**Comment by radioactiveman** [comment]
ReplayGain is not supported yet by the FFmpeg plugin of Audacious.
At least I didn't found any references for it in the [source code](https://github.com/audacious-media-player/audacious-plugins/tree/master/src/ffaudio).

@jlindgren90: Can you confirm this?

**Reply by jlindgren90** [reply]
Sounds right

**Reply by Lascaille** [reply]
If I can figure out how to implement it, would you merge? (I'm not confident, but I'll look.)

**Reply by jlindgren90** [reply]
Anyone is welcome to submit a pull request. We don't make promises in advance about what will be merged. It depends on the quality of the code.
